tests/test_operators/test_is_unique_set.py

```diff
@@ -0,0 +1,101 @@
+from business_rules.operators import DataframeType
```

Comment:
These tests can probably go in the test_dataframe_type folder, since it holds tests for all of the dataframe type operators